src/nni_manager/rest_server/restHandler.ts

```diff
@@ -168,13 +169,20 @@ class NNIRestHandler {
                     // Start experiment is a step of initialization, so any exception thrown is a fatal
                     this.handle_error(err, res);
                 });
-            } else {
+            } else if (experimentMode === 'resume'){
                 this.nniManager.resumeExperiment().then(() => {
                     res.send();
                 }).catch((err: Error) => {
                     // Resume experiment is a step of initialization, so any exception thrown is a fatal
                     this.handle_error(err, res);
                 });
+            } else {
```

Comment:
It's better to use else if (experimentMode === 'view'){

Reply:
updated